src/runtime/ipx.ts

```diff
@@ -1,16 +1,10 @@
-import { fileURLToPath } from 'node:url'
 import { createIPX, createIPXMiddleware } from 'ipx'
 import { withLeadingSlash } from 'ufo'
 import { eventHandler, lazyEventHandler } from 'h3'
 import { useRuntimeConfig } from '#imports'
 
 export default lazyEventHandler(() => {
-  const ipxOptions = {
-    ...(useRuntimeConfig().ipx || {}),
-    // TODO: Switch to storage API when ipx supports it
-    dir: fileURLToPath(new URL('../public', import.meta.url))
```

Comment:
This workaround works also for the production `.output/` directory structure  as path is resolved at runtime. If we do `resolve()` in `src/ipx.ts`, full path of build environment will be into the bundle.